src/main/java/Order/LottoOrder.java

```diff
@@ -0,0 +1,20 @@
+package Order;
+
+public class LottoOrder extends Order{
+    private final int manualTicketCounts;
+    public LottoOrder(int money, int manualTicketCounts) {
```

Comment:
`합성` 보다 `상속`으로 구현하신 이유가 궁금해요! 🤔 
개인적으론 합성이 더 적합하기도 하고 쉽지 않을까 생각이 되어서요!

[[OOP] 코드의 재사용, 상속(Inheritance)보다 합성(Composition)을 사용해야 하는 이유](https://mangkyu.tistory.com/199)
```suggestion
    public LottoOrder(Money money, int manualTicketCounts) {
```